src/history/html5.js

```diff
@@ -87,7 +87,9 @@ export class HTML5History extends History {
 
 export function getLocation (base: string): string {
   let path = window.location.pathname
-  if (base && path.toLowerCase().indexOf(base.toLowerCase()) === 0) {
```

Comment:
Let's create two variables `pathLowerCase` and `baseLowerCase`

Reply:
@posva done